test/JuMPExtension.jl

```diff
@@ -51,11 +51,15 @@ function JuMP.add_variable(m::MyModel, v::JuMP.AbstractVariable, name::String=""
     JuMP.set_name(vref, name)
     vref
 end
-function MOI.delete!(m::MyModel, vref::MyVariableRef)
-    delete!(m.variables, vref.idx)
-    delete!(m.varnames, vref.idx)
+function JuMP.delete(model::MyModel, variable_ref::MyVariableRef)
+    @assert JuMP.is_valid(model, variable_ref)
```

Comment:
Nit: Error message instead of `assert`.

Reply:
I think I just made it an assert because it is a test for an extension. Should this be a style thing to never use asserts and always have an informative error message?

Reply:
using errors in JuMPExtension is maybe overkill

Reply:
Oops, I got confused. It's fine to use this in a test.